src/core/components/message/message.scss

```
@@ -45,6 +45,7 @@
         .main {
             padding: 8px;
             flex-grow: 1;
+            overflow: hidden;
```

```suggestion
            overflow: hidden;
            text-overflow: ellipsis;
            white-space: nowrap;
```

Maybe this would be better, so that the cut is not as abrupt? This is using the "truncate" technique, for example as documented in TailwindCSS: https://tailwindcss.com/docs/text-overflow#truncate

---

And I just noticed we already use it in some places with the class `text-truncate`.